simapp/go.mod

```diff
@@ -196,6 +196,7 @@ replace (
 	cosmossdk.io/x/evidence => ../x/evidence
 	cosmossdk.io/x/feegrant => ../x/feegrant
 	cosmossdk.io/x/nft => ../x/nft
+	cosmossdk.io/x/tx => ../x/tx
```

Comment:
It seems like the liveness-test is failing because of this change:
```
go: cosmossdk.io/x/tx@v0.2.0 (replaced by ./x/tx): reading x/tx/go.mod: open /work/x/tx/go.mod: no such file or directory
```

any ideas how I can fix it?

Reply:
I believe in one of the dockerfile?

Reply:
I tried [60e1d95](https://github.com/cosmos/cosmos-sdk/pull/15278/commits/60e1d95b45c1bff1ac892090fadc3c5236893e1e), let's see